Core/VariantManager.swift

```diff
@@ -25,34 +25,49 @@ public enum FeatureName: String {
     // Used for unit tests
     case dummy
     
+    case iPadImprovements
     case tabSwitcherListLayout
     
 }
 
-public enum CohortFiltering {
+public enum VariantInclusion {
     
-    case includeInCohort
-    case excludeFromCohort
+    case always
+    case when(_ filter: () -> Bool)
     
 }
 
 public struct Variant {
     
+    struct Is {
+        static let padDevice = { return UIDevice.current.userInterfaceIdiom == .pad }
+        static let notPadDevice = { return !Is.padDevice() }
+    }
+    
     static let doNotAllocate = 0
     
+    // Note: Variants with `doNotAllocate` weight, should always be included so that previous installations are unaffected
     public static let defaultVariants: [Variant] = [
+        
         // SERP testing
-        Variant(name: "sc", weight: 1, features: []),
-        Variant(name: "sd", weight: doNotAllocate, features: []),
-        Variant(name: "se", weight: 1, features: []),
+        Variant(name: "sc", weight: 1, features: [], isIncluded: .when(Is.notPadDevice)),
```

Comment:
Could we move the `isIncluded` variable to be after the weight? Seems more natural to have the weight and filtering next to each other.